internal/app/synchronizer/synchronizer_service.go

```diff
@@ -396,23 +402,44 @@ func (s *synchronizerService) wrapEvaluator(ctx context.Context, cancel cancelEr
 	close(m4c)
 }
 
+func (s *synchronizerService) cacheMatchIDToTicketIDs(m *sync.Map, m3c <-chan []*pb.Match, m3point5c chan<- *pb.Match) {
```

Comment:
should this be in a comment section (/////////////////////////////////////////////)